### DISCUSSION TITLE
talk abt stuff that isnt actually an issue here

### DISCUSSION THREAD

**Discussion by zek-c** [zek-c/Securly-Kill-V111#50] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
(yap session)
  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by local-slav** [comment]
im nuking the yapaneses

**Reply by hueychen27** [reply]
unique introduction

**Comment by dragon731012** [comment]
@zek-c https://discord.com/channels/419123358698045453/1198552768542167130

**Reply by zek-c** [reply]
what i'm not in that server i can't see that

**Reply by dragon731012** [reply]
here typed up some code to make an easy way to disable securly https://github.com/dragon731012/caudns

**Reply by dragon731012** [reply]
took only an hour lol

**Reply by dragon731012** [reply]
https://caudns.vercel.app/

**Reply by zek-c** [reply]
that's actually fire, wanna make a pr to add it to this repo? make a lil writeup in the readme and add it in its own folder and create a html file in the root dir 👍

**Reply by dragon731012** [reply]
Can you pls I’m too lazy

**Reply by zek-c** [reply]
sure lmao 😭‼️🙏

**Reply by dragon731012** [reply]
Ty man

**Reply by dragon731012** [reply]
Might add more stuff so I’ll keep this repo updated too

**Reply by dragon731012** [reply]
also can I have collaborator

**Reply by dragon731012** [reply]
trust

**Reply by zek-c** [reply]
i don't see why not, you in fact have collaborated on this and you've been here since the start

**Comment by goblergoofy** [comment]
how do i do this

**Reply by zek-c** [reply]
read the readme.md file duh

**Comment by sevenlesshope** [comment]
> thats a policy block which cant be bypassed GG 😭

luckily my settings crashed and burned whilst i had an old DNS unblocker so at least i can still have...like discord and ponytown. sigh.

i mean, maybe by some force of nature they'll give me settings perms back but i doubt it. honestly surprised they haven't like put me on probation or something for the amount of unblockers/loopholes i've found in this swiss cheese of an operating system lmao??

(i lied, i literally just looked around github for like 3 minutes. or youtube. or tiktok. also, is there a discord server?)

**Reply by sevenlesshope** [reply]
thats a policy block which cant be bypassed GG 😭

YOOO it turns out the dns that saved was acutally the caudns one, so my computer and my school admin are in a stalemate 💀 the computer is technically permanently unblocked (unless the dns is patched which i doubt it can be) until they give back settings app perms + perms to view the chrome://network website, meaning i cant manually turn it off or on

only problem is with tests, because i dont think hapara works uh
pretty sure i can fix it by just connecting to a hotspot which would allow hapara to have perms on my computer again (i didn't upload the kill.onc to my phone hotspot network) unless hapara doesnt work with hotspots.

if it doesnt im a sitting duck bro

edit: i dont think it does. im cooked. 😢 

(i'll take any help i can get uh)

**Reply by sevenlesshope** [reply]
i can probably make excuses until the end of the year but idk if my teachers are THAT dumb.. depends on if they can view my tabs in hapara or not. if they cant thats kind of a dead giveaway while i'm doing tests but oh well, guess i flew too close to the sun

**Reply by zek-c** [reply]
if you can view chrome://network you can manually config a DNS to use

**Reply by sevenlesshope** [reply]
i cant uh

**Reply by sevenlesshope** [reply]
i got the settings app back, anything i can do?

**Reply by sevenlesshope** [reply]
OH MY LORD I CAN EDIT ONE OF THE SCHOOL NETWORKS' DNS I'M SAVED. and it's the one with the nice wifi too RAGHHH

**Comment by sevenlesshope** [comment]
probably a really dumb suggestion but
i noticed whenever you're on the "new tab" chrome page the little securly extension thing is gray (which i guess means it's off?)

i guess i was just wondering if there was enythijg that could be done with that? unless it's impossible or would have to use bookmarklets (which, iirc, don't work in newer updates unless caub is used)

**Comment by SchoolTARMaster** [comment]
insecurly link: https://mathqualifiedtests.vercel.app

**Reply by zek-c** [reply]
ok

**Comment by Supermon7** [comment]
why cant i click the download button in cauDNS? i did all the steps it just wont let me click it

**Reply by zek-c** [reply]
you must've done it wrong, the download button only works when a correct wifi config is input

**Reply by Supermon7** [reply]
yeah i did it wrong. it sucks becuase chrome://network#state and basically all the chrome:// websites are blovked by admidistrator not even securly so i cant really do anything ab it :(

**Reply by Supermon7** [reply]
do u know if there is a way i can get on them still even tho admin/organization blocked it?

**Reply by zek-c** [reply]
if chrome://network is blocked u cant upload the .onc file
